backends/graphics/surfacesdl/surfacesdl-graphics.cpp

```diff
@@ -2921,19 +2934,16 @@ void SurfaceSdlGraphicsManager::SDL_UpdateRects(SDL_Surface *screen, int numrect
 		viewport.y = drawRect.left + delta;
 	}
 	rotangle = rotation;
-#endif
+
 	viewport.w = drawRect.width();
 	viewport.h = drawRect.height();
 
 	SDL_RenderClear(_renderer);
-#if SDL_VERSION_ATLEAST(2, 0, 0)
```

Comment:
Also, it is not clear, why is this now removed

Reply:
Same as above. It's in the same commit.